### DISCUSSION TITLE
Lighthouse V1 angles

### DISCUSSION THREAD

**Discussion by apark2459** [bitcraze/discussions#1508] [Q&A] [answered]
Hello,

I am trying to estimate the basestation geometry using the lighthouse_bs_geo.py script (https://github.com/bitcraze/crazyflie-lib-python/blob/master/cflib/localization/lighthouse_bs_geo.py) but am having little success. Is it possible at all to get an accurate pose estimation for the basestation using just lighthouse_bs_geo.py or would I have to make use of the other scripts like lighthouse_geometry_solver.py (https://github.com/bitcraze/crazyflie-lib-python/blob/master/cflib/localization/lighthouse_geometry_solver.py)? (FYI: I am trying to interface it with a RPi and so far I have successfully adapted the pulse_processor_v2.c code and am getting stable angle measurements with little noise)

Thanks!

**Comment by apark2459** [comment]
Or is it possible that the basestation pose is off by a constant scaling factor since focal lengths of 1 was used?

**Comment by gemenerik** [comment, marked as answer]
Please try the `lighthouse_geometry_solver.py` and let me know if it works

**Reply by apark2459** [reply]
I will try that. Thank you!

**Reply by apark2459** [reply]
It turns out it was another part of my code that was making it inaccurate. I am getting good accuracy with just lighthouse_bs_geo.py alone!

**Reply by gemenerik** [reply]
Good to hear, thanks for the update!
